packages/yourloops/test/__mocks__/@auth0/auth0-react.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line no-undef
```

Comment:
Have you tried removing these?

Reply:
Don't remember... but this file no longer exists now